modules/eks/iam.tf

```diff
@@ -277,7 +277,7 @@ resource "aws_eks_identity_provider_config" "this" {
     groups_claim                  = lookup(each.value, "groups_claim", null)
     groups_prefix                 = lookup(each.value, "groups_prefix", null)
     identity_provider_config_name = each.value.identity_provider_config_name
-    issuer_url                    = try(each.value.issuer_url, aws_iam_openid_connect_provider.oidc_provider.url)
```

Comment:
Is the `identity_providers` value `issuer_url` ignored now?

Reply:
No,good catch